js/private/js_library.bzl

```diff
@@ -196,7 +196,7 @@ def _js_library_impl(ctx):
     )
 
     npm_package_store_infos = gather_npm_package_store_infos(
-        targets = ctx.attr.data + ctx.attr.deps,
+        targets = ctx.attr.srcs + ctx.attr.data + ctx.attr.deps,
```

Comment:
`srcs` was notably missing here. this is a related bug fix

Reply:
I was wondering about that. Was this required for this PR? Should this go into 1.x?

Reply:
It could but it's a rare corner case that no one has hit yet so may not be worth the effort

Reply:
split it out to https://github.com/aspect-build/rules_js/pull/1650